shoop_tests/utils/test_enum.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from shoop.core.models.products import ProductMode
+from shoop.utils.enums import map_enum
+
+
+def test_map_enum():
+    assert map_enum(ProductMode, 0) is ProductMode.NORMAL
```

Comment:
The point is that it is _able_ to do that mapping (which I foresee being useful for, say, import/export situations), but user code probably shouldn't use magic numbers.